.github/workflows/nodejs.yml

```diff
@@ -39,6 +39,7 @@ jobs:
     - run: yarn build
     - run: yarn format:ci
     - run: yarn eslint
+    - run: yarn run jsdoc prepare
```

Comment:
This will always fail as the dependencies are not installed. I've fixed this in #1701 already, can just remove?